app/src/main/res/values-fr/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
I don't quite understand that—could you clarify it for me, please? I think it was unused, right?

Reply:
Nice catch. I have no idea how this happened. I'm confident I haven't touched strings.xml outside merges.

Reply:
Turns out it was unused. It must have been removed automatically by optimize imports or something.